platform/ios/src/MGLMapView.mm

```diff
@@ -390,6 +393,8 @@ - (void)commonInit
 {
     MGLinitializeRunLoop();
 
+    _isWaitingForStyleLoad = YES;
```

Comment:
Fixed in https://github.com/mapbox/mapbox-gl-native/pull/7639/commits/80f96a1e5519c62ff3afd3dab112866d4ba1be41